LambdaFactory/Fornax
# Live refresh

This adds live refresh to the `fornax watch` command.  It can be disabled via `fornax watch --disable-live-refresh`.

![fornax_600](https://user-images.githubusercontent.com/1795462/69467341-869c6880-0d55-11ea-99d7-e6864476f1d5.gif)

Sorry the gif is so tiny, it's the biggest one I could upload.

How to test:

1. Clone repo
2. `cd src/Fornax`
3. `./resetTool.sh`
4. `mkdir test`
5. `cd test`
6. `fornax new`
7. `fornax watch`
8. Wait until it says it's done, then navigate to http://localhost:8080/ .
9. Make a change to one of the `*.md` files.
10. See the page reload in your browser.

There's only one annoying thing left, it's not really a big deal but I'm not sure how to get rid of it.  When the page reloads, Suave outputs the following message:

<img width="881" alt="image" src="https://user-images.githubusercontent.com/1795462/69467464-16421700-0d56-11ea-8337-2a64bf569029.png">

I'm sure there's a way to get rid of it, I'm just not sure how.

Let me know if any changes should be made.
